README.md

```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```

Comment:
Maybe?

Could perhaps link to https://packaging.python.org/installing/#install-pip-setuptools-and-wheel